go-router/main/default/pages/go_setup.page-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexPage xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
Does there need to be a tab for this as well? Or is just a Visualforce page sufficient?

Reply:
A tab would be ideal for easily returning to the page. Is there specific setup to make it a tab?